pkg/reconciler/common/prune.go

```diff
@@ -393,19 +391,25 @@ func createCronJob(ctx context.Context, kc kubernetes.Interface, cronName, targe
 // ns-one;--keep=5;pipelinerun
 // ns-two;--keep=3;pipelinerun,taskrun
 // ns-three;--keep=2;taskrun
+// ns-four;--keep=4 --keep-since=300; pipelinerun
 // these configs are passed as space seperated string argument to pod container spec
 // for each of this namespaced config below commands are generated
 // tkn pipelinerun delete --keep=5 -n=ns-one -f ;
 // tkn pipelinerun delete --keep=3 -n=ns-two -f ;
 // tkn taskrun delete --keep=3 -n=ns-two -f ;
 // tkn taskrun delete --keep=3 -n=ns-three -f ;
+// tkn pipelinerun delete --keep=4 --keep-since=300 -n=ns-four -f ;
 func generatePruneConfigPerNamespace(pru *pruneConfigPerNS, ns string) string {
 	var keepCmd string
```

Comment:
This can be simplified like

```
keepCmd := "" 
if pru.config.Keep != nil {
	keepCmd = keepCmd + " --keep=" + fmt.Sprint(*pru.config.Keep)
}
if pru.config.KeepSince != nil {
	keepCmd = keepCmd + " --keep-since=" + fmt.Sprint(*pru.config.KeepSince)
}
```

Is extra space creating an issue?

Reply:
IIRC I did it like this the first time, but somehow it was not working, so I changed. do you want me to check again? I might have missed it.